### PR TITLE
Fixed #32371 -- Doc'd jquery.init.js dependency for admin widgets.

### PR DESCRIPTION
**Pull request by matthiask**
Until now this was only documented in the 2.2 release notes. See https://github.com/django/django/commit/418263c4576

I think it's worth it documenting this so people can rely on this.

### CODE REVIEW

File: docs/ref/contrib/admin/index.txt [hunk 1]
```diff
@@ -2228,7 +2228,9 @@ Django admin JavaScript makes use of the `jQuery`_ library.
 To avoid conflicts with user-supplied scripts or libraries, Django's jQuery
 (version 3.5.1) is namespaced as ``django.jQuery``. If you want to use jQuery
 in your own admin JavaScript without including a second copy, you can use the
-``django.jQuery`` object on changelist and add/edit views.
+``django.jQuery`` object on changelist and add/edit views. If you want to use
+``django.jQuery`` in your own admin forms or widgets you must specify the
+dependency as follows: ``js=['admin/js/jquery.init.js', ...]``.
```
Comment (felixxm) [inline docs/ref/contrib/admin/index.txt:2233]: Maybe:
```suggestion
``django.jQuery`` object on changelist and add/edit views. Also, your own admin
forms or widgets depending on ``django.jQuery`` must specify
``js=['admin/js/jquery.init.js', …]` when :ref:`declaring form media assets
<assets-as-a-static-definition>`.
```

File: docs/ref/contrib/admin/index.txt [hunk 2]
```diff
@@ -2228,7 +2228,10 @@ Django admin JavaScript makes use of the `jQuery`_ library.
 To avoid conflicts with user-supplied scripts or libraries, Django's jQuery
 (version 3.5.1) is namespaced as ``django.jQuery``. If you want to use jQuery
 in your own admin JavaScript without including a second copy, you can use the
-``django.jQuery`` object on changelist and add/edit views.
+``django.jQuery`` object on changelist and add/edit views. Also, your own admin
+forms or widgets depending on ``django.jQuery`` must specify
+``js=['admin/js/jquery.init.js', …]` when :ref:`declaring form media assets
```
Comment (felixxm) [inline docs/ref/contrib/admin/index.txt:2233]:
```suggestion
``js=['admin/js/jquery.init.js', …]`` when :ref:`declaring form media assets
```
I missed a backtick.

**Comment by matthiask** [comment]
Thanks for your improvements!

**Comment by felixxm** [comment]
@matthiask Thanks :+1:

**Pull request merged**